submissions/korotushak96/js-dom/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="index.css">
+    <title>Fantastic Beasts</title>
+</head>
+<body>
+    <h1>Fantastic Beasts</h1>
+    <div class="burger">
```

Comment:
Also try to use HTML5 tags like `aside`, `section`, `figure`, `main`, `nav` etc.
And don't forget about [WAI ARIA](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/WAI-ARIA_basics) in your markup. For example, in your burger button you can add an atrribute `role='button'` etc.

Reply:
Ok. I will use it in future.